packages/react/src/component-tests/IcPageHeader/IcPageHeader.cy.tsx

```diff
@@ -0,0 +1,133 @@
+/// <reference types="Cypress" />
+
+import React from "react";
+
+import { mount } from "cypress/react";
+import { BE_VISIBLE, HAVE_LENGTH } from "../utils/constants";
+import {
+  DesktopScrollablePageVariant,
+  PageHeadeWithBreadcrumbNav,
+  PageHeaderDefault,
+  PageHeaderSmallSize,
+  PageHeaderWithActionsInputTabs,
+  PageHeaderWithStepper,
+  PageHeaderWithoutBorder,
+  PageheaderAlign,
+  ScrollStickyPageHeader,
+} from "./IcPageHeaderTestData";
+import { setThresholdBasedOnEnv } from "../../../cypress/utils/helpers";
+
+const DEFAULT_TEST_THRESHOLD = 0.05;
+
+describe("IcPageHeader e2e, A11y and visual regression tests", () => {
+  beforeEach(() => {
+    cy.injectAxe();
+    cy.viewport(1450, 500);
+  });
+
+  afterEach(() => {
+    cy.task("generateReport");
+  });
+
+  it("render default PageHeader", () => {
+    mount(<PageHeaderDefault />);
+    cy.checkHydrated("ic-page-header");
+    cy.checkA11yWithWait();
+    cy.compareSnapshot({
+      name: "default",
+      testThreshold: setThresholdBasedOnEnv(0.02),
+    });
+  });
+
+  it("should stay at the top of the page if the sticky variant is used", () => {
+    cy.viewport(1050, 450);
+    mount(<ScrollStickyPageHeader />);
+    cy.checkA11yWithWait();
+    cy.scrollTo("bottom").checkHydrated("ic-button#end");
+    cy.checkA11yWithWait();
+    cy.compareSnapshot({
+      name: "stickyVariant",
+      testThreshold: setThresholdBasedOnEnv(0.02),
+    });
+  });
+
+  it("page header should not stay on top if prop use stickyDesktopOnly for viewport extra small", () => {
+    mount(<DesktopScrollablePageVariant />);
+    cy.viewport("iphone-6");
+    cy.compareSnapshot({
+      name: "desktopOnlyBeforeScroll",
+      testThreshold: setThresholdBasedOnEnv(0.02),
+    });
+    cy.scrollTo(0, 800);
+    cy.compareSnapshot({
+      name: "desktopOnlyAfterScroll",
+      testThreshold: setThresholdBasedOnEnv(0.02),
+    });
+  });
```

Comment:
Solution for this sticky page header screenshot
```suggestion
  it("page header should not stay on top if prop use stickyDesktopOnly for viewport extra small", () => {
    cy.viewport("iphone-6")
    mount(<DesktopScrollablePageVariant />);
    cy.compareSnapshot({
      name: "desktopOnlyBeforeScroll",
      testThreshold: setThresholdBasedOnEnv(0.02),
      cypressScreenshotOptions: {
        capture: 'viewport'
      }
    });
    cy.scrollTo("bottom");
    cy.compareSnapshot({
      name: "desktopOnlyAfterScroll",
      testThreshold: setThresholdBasedOnEnv(0.02),
      cypressScreenshotOptions: {
        capture: 'viewport'
      }
    });
  });
```